doc/backend/branding.md

```diff
@@ -136,3 +136,89 @@ like `@page`. Supported css features depending on weasyprint, see [here](https:/
 
 The second one (`rich_text_preview.template`) is a templating file of html for pandoc, see [here](https://pandoc.org/MANUAL.html#template-syntax)
 for specific template syntax.
+
+
+## App Custom Action
+
+In content apps (File, Thread, Note, Kanban, Logbook), there is a kebab menu accessible by the button "⋮".
+
+App Custom Action allows to add submenu options with custom link.
+
+### Create an App Custom Actions
+
+Update the file `frontend/dist/assets/branding/app_custom_actions.json`.
+
+It is a json file containing a list of objects. Each objects are the definition of an App Custom Actions.
+
+**Available parameters:**
+- `icon_text`:
+  - The icon preceding the submenu option. Use it for icon from css library.
+  - Example: `"icon_text": "fas fa-wine-bottle"`
+- `icon_image`:
+  - The image preceding the submenu option. Use it for images from an online source.
+  - If icon_text and icon_image are set, only image will be displayed.
+  - Example: `"icon_image": "https://www.tracim.fr/static/images/new_tracim/LOGO_TRACIM_RVB_1.png"`
```

Comment:
is it not possible to set an url of local tracim ?

do we use tracim.fr website in another example ? Or maybe just use not real domain but leave this path.

Reply:
Or better, use a link with tracim github repository :)

Reply:
I wanted to give an absolute url as an example, I think using one from github is a good idea indeed